src/main/kotlin/no/nav/syfo/identhendelse/database/IdenthendelseQuery.kt

```diff
@@ -0,0 +1,114 @@
+package no.nav.syfo.identhendelse.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.domain.Personident
+import java.sql.Connection
+import java.sql.PreparedStatement
+
+const val queryUpdateBehandler =
+    """
+        UPDATE BEHANDLER
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun Connection.updateBehandler(
+    nyPersonident: Personident,
+    inactiveIdenter: List<Personident>,
+    commit: Boolean = false,
+): Int {
+    return this.updateIdent(
+        query = queryUpdateBehandler,
+        nyPersonident = nyPersonident,
+        inactiveIdenter = inactiveIdenter,
+        commit = commit,
+    )
+}
+
+const val queryBehandlerArbeidstaker =
+    """
+        UPDATE BEHANDLER_ARBEIDSTAKER
+        SET arbeidstaker_personident = ?
+        WHERE arbeidstaker_personident = ?
+    """
+
+fun Connection.updateBehandlerArbeidstaker(
+    nyPersonident: Personident,
+    inactiveIdenter: List<Personident>,
+    commit: Boolean = false,
+): Int {
+    return this.updateIdent(
+        query = queryBehandlerArbeidstaker,
+        nyPersonident = nyPersonident,
+        inactiveIdenter = inactiveIdenter,
+        commit = commit,
+    )
+}
+
+const val queryUpdateBehandlerDialogmeldingBestilling =
+    """
+        UPDATE BEHANDLER_DIALOGMELDING_BESTILLING
+        SET arbeidstaker_personident = ?
+        WHERE arbeidstaker_personident = ?
+    """
+
+fun Connection.updateBehandlerDialogmeldingBestilling(
+    nyPersonident: Personident,
+    inactiveIdenter: List<Personident>,
+    commit: Boolean = false,
+): Int {
+    return this.updateIdent(
+        query = queryUpdateBehandlerDialogmeldingBestilling,
+        nyPersonident = nyPersonident,
+        inactiveIdenter = inactiveIdenter,
+        commit = commit,
+    )
+}
+
+private fun Connection.updateIdent(
```

Comment:
😃 Smart med egen funksjon for dette

Reply:
![image](https://user-images.githubusercontent.com/37441744/211018171-4cd57f91-0e4c-4c05-84cc-6803efd9d21e.png)